### PR TITLE
Switch the kitchen tests to Policyfiles

### PR DESCRIPTION
**Pull request by tas50**
Test what we recommend

Signed-off-by: Tim Smith <tsmith@chef.io>

### CODE REVIEW

**Review by lamont-granquist — APPROVED** [review]

File: .github/workflows/kitchen.yml [hunk 1]
```diff
@@ -16,15 +16,10 @@ jobs:
...
-      id: install_chef
-      run: |
-        . { Invoke-WebRequest -useb https://omnitruck.chef.io/install.ps1 } | Invoke-Expression; Install-Project -project chef -channel current
-        $env:PATH = "C:\opscode\chef\bin;C:\opscode\chef\embedded\bin;" + $env:PATH
-        chef-client -v
-        ohai -v
-        rake --version
-        bundle -v
+    - name: Install Infra Client from current channel
+      uses: actionshub/chef-install@master
```
Comment (damacus) [inline .github/workflows/kitchen.yml:20]:
```suggestion
      uses: actionshub/chef-install@main
```

actionshub has switched to the main branch now and @Xorima is looking to remove the master branch shortly 👍🏼

**Review by damacus — CHANGES_REQUESTED** [review]
master --> main branch for actionshub chef-installs

File: .github/workflows/kitchen.yml [hunk 2]
```diff
@@ -60,15 +50,10 @@ jobs:
...
-      id: install_chef
-      run: |
-        brew install coreutils
-        curl -L https://omnitruck.chef.io/install.sh | sudo bash -s -- -c current
-        /opt/chef/bin/chef-client -v
-        /opt/chef/bin/ohai -v
-        /opt/chef/embedded/bin/rake --version
-        /opt/chef/embedded/bin/bundle -v
+    - name: Install Infra Client from current channel
+      uses: actionshub/chef-install@master
```
Comment (Xorima) [inline .github/workflows/kitchen.yml:54]:
```suggestion
      uses: actionshub/chef-install@main
```

**Review by Xorima — CHANGES_REQUESTED** [review]

**Review by damacus — APPROVED** [review]

**Review by Xorima — APPROVED** [review]

**Review by lamont-granquist — APPROVED** [review]

**Comment by Xorima** [comment]
Leave the windows side with me and I will try to get a fix in for it

**Pull request closed without merging**